src/Adapter/Product/AdminProductWrapper.php

```diff
@@ -268,7 +268,7 @@ public function processDependsOnStock(Product $product, $dependsOnStock, $forAtt
     public function processProductSpecificPrice($id_product, $specificPriceValues, $idSpecificPrice = null)
     {
         // ---- data formatting ----
-        $id_product_attribute = $specificPriceValues['sp_id_product_attribute'];
+        $id_product_attribute = $specificPriceValues['sp_id_product_attribute'] ?? 0;
         $id_shop = $specificPriceValues['sp_id_shop'] ? $specificPriceValues['sp_id_shop'] : 0;
         $id_currency = $specificPriceValues['sp_id_currency'] ? $specificPriceValues['sp_id_currency'] : 0;
         $id_country = $specificPriceValues['sp_id_country'] ? $specificPriceValues['sp_id_country'] : 0;
```

Comment:
```suggestion
        $id_shop = $specificPriceValues['sp_id_shop'] ?? 0;
        $id_currency = $specificPriceValues['sp_id_currency'] ?? 0;
        $id_country = $specificPriceValues['sp_id_country'] ?? 0;
```

Not blocking the PR but, should be more consistent :thinking: use the same way everywhere :sweat_smile: 

With :heart:

Reply:
Let's do this together 💪

Reply:
Good evening @matks 
Could you do the same on the line 275 please. 😅
Thank you man

Reply:
And when I do it, someone else will pop up and ask me to do it again on another line ? 😛 is that your plan ?

Reply:
"Make prestashop consistent again ..." 😅

Reply:
(Luigi's voice) "Here we go !"
https://github.com/PrestaShop/PrestaShop/pull/19845/commits/12f779e467f7a066a40a2f2155a1609068581c99